tools/api-markdown-documenter/README.md

```diff
@@ -144,28 +144,54 @@ As noted above, this library is intended to be consumed programmatically.
 While we may at some point add some command line interfaces for common paths, the primary goal of this library's architecture is to be flexible and extensible.
 To that end, we have broken its logic into a few isolable steps, each offering its own extensibility offerings.
 
-End to end, this library can be viewed as a psuedo-functional transformation pipeline mapping from an [APIModel][] generated by `API-Extractor` to one or more `Markdown`-formatted document files.
+End to end, this library can be viewed as a psuedo-functional transformation pipeline mapping from an [APIModel][] generated by `API-Extractor` to one or more `Markdown` or `HTML` formatted document files.
+
 But this is broken into the following internal sequences:
 
 ```mermaid
+
+---
+title: Markdown
+---
+
 graph LR
     A[ApiModel]
     B[Documentation AST]
-    C[raw Markdown]
+    C[Markdown AST~]
+    D[raw Markdown]
+
+    A-->|transformApiModel|B
+    B-->|documentToMarkdown~|C
```

Comment:
I really like how the HAST domain intermediary has worked out (and it aligns with the original intentions for this library). I'm hoping to get the MDAST side of things in the near future (probably on my own time, because I'm a nerd).